01-messaging-crypto-and-init.md

```diff
@@ -20,24 +20,21 @@ the unicode code point for LIGHTNING.<sup>[2](#reference-2)</sup>
 
 All messages are of form:
 
-1. `4-byte` big-endian data length.
-2. `4-byte` big-endian type.
-3. Data bytes as specified by the length.
+1. `2-byte` big-endian type.
```

Comment:
@cdecker Yes, I like this idea.  Add 2 bytes to the framing header, meaning a naive implementation will have this naturally-aligned (a decrypt-in-place will be 4-byte aligned, but will anyone really do that?).

What does @Roasbeef think?